minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloConfigImpl.java

```diff
@@ -56,6 +56,7 @@
   private Map<String,String> configuredSiteConig = new HashMap<>();
   private Map<String,String> clientProps = new HashMap<>();
   private int numTservers = 2;
+  private int numScanServers = 2;
```

Comment:
This will make Mini Accumulo all of a sudden use more memory for an optional feature.  Wondering if this should default to zero.

Reply:
Fixed in 7521ae6